packages/cli/test/integration/api-keys.api.test.ts

```diff
@@ -30,7 +29,8 @@ beforeAll(() => {
 
 beforeEach(async () => {
 	await testDb.truncate(['User']);
-	mockInstance(GlobalConfig, { publicApi: { disabled: false } });
+	globalConfig = Container.get(GlobalConfig);
```

Comment:
@guillaumejacquart please help me understand how the changes in this file are related to the auth cookie name change?

Reply:
Makes sense, thank you